pkg/driverbuilder/docker.go

```diff
@@ -259,6 +259,7 @@ func (bp *DockerBuildProcessor) Start(b *builder.Build) error {
 		Env:          envs,
 		Cmd: []string{
 			"/bin/bash",
+			"-l",
```

Comment:
By setting the "login" flag to bash, we enforce it to load data from bashrc/bash profiles.
See https://stackoverflow.com/questions/55206227/why-bashrc-is-not-executed-when-run-docker-container.